float32  position_x 
float32  position_y
float32  position_z
float32  orientation_x
float32  orientation_y
float32  orientation_z
float32  orientation_w

float32  angle_x
float32  angle_y
float32  angle_z

float32 translation_speed
float32 orientation_speed
